hack/cluster-debugging-scripts/README.md

```diff
@@ -0,0 +1,55 @@
+# Cluster Debugging Scripts
+
+This folder contains a collection of scripts to help debug clusters.
+
+## FAQ
+
+### How can I validate that the aws-ebs-csi-driver correctly makes use of all available attachment slots for instance type X? 
+
+Answer: Perform the following steps (Which will create a nodegroup, count the Block Device Mappings + ENI for the underlying instance, deploy pods EBS PVs until the script finds the maximum amount of volumes the aws-ebs-csi-driver can attach to instance)
+
+```
+export CLUSTER_NAME="devcluster"
+export NODEGROUP_NAME="ng-attachment-limit-test"
+export INSTANCE_TYPE="m7g.large"
+
+eksctl create nodegroup -c "$CLUSTER_NAME" --nodes 1 -t "$INSTANCE_TYPE" -n "$NODEGROUP_NAME"
+
+./get-attachment-breakdown "$NODEGROUP_NAME" 
```

Comment:
Is there a reason the script uses a nodegroup here? Wouldn't just supplying the instance type and using the CCM instance type label be better?

Reply:
Originally it was done because nodegroup names are less likely to clash than instance types, but you brought up the point that this choice makes the script useless on kOps clusters. 

Will change to instance type.